aviary/subsystems/propulsion/gearbox/model/gearbox_premission.py

```diff
@@ -56,7 +69,7 @@ def setup(self):
                     has_diag_partials=True,
                 ),
                 promotes_inputs=[
-                    'torque_max',
+                    # 'torque_max',
```

Comment:
I see a few inputs commented out. Are these variables now connected directly by the multiengine demux?

Reply:
They are still connected within this component, just manually with `self.connect()`. This was done a while ago so I'm not sure which exact issue required this, but it may be that letting `torque_max` get promoted was causing problems downstream into mission or somewhere else.